vcf/filter.go

```diff
@@ -209,6 +209,32 @@ func IsPolarizable(v Vcf) bool {
 	return true
 }
 
+//IsWeakToStrong returns true if an input biallelic substitution variant is a weak to strong variant, false otherwise.
+func IsWeakToStrong(v Vcf) bool {
+	if !IsBiallelic(v) || !IsSubstitution(v) {
+		return false
+	}
+	if v.Ref == "A" || v.Ref == "T" {
```

Comment:
Don't we also need to check the ancestral allele in this function?  Seems like we need the ancestral allele to be A/T and the derived allele (could be ref or alt) to be G/C.

Reply:
This could go either way. If we are worried that we will be biased for weak to strong mutations in the derived state, we can overestimate positive selection with high frequency WtS mutants. But conversely, if there is a corresponding bias against StW variants, we may also want to remove those or risk overestimating negative selection. I'm not sure what's best

Reply:
I understand.  While this may be what you want for your research question.  I think it might cause confusion if someone has their G->A mutations removed when they say they want to remove "weak to strong" mutations.  Maybe we want to change the name, or to implement both IsWeakToStrong and IsStrongToWeak and then you will give both those flags for your research.

Reply:
If what I'm inferring about the definition/use of "strong to weak" etc to be true, then I do agree that it would make sense to me to break it up. I'm thinking lines 217 to 221 are referring to weak to strong and lines 222 to 228 are strong to weak... assuming that ref is ancestral since there is directionality in the way this is interpreted? Assuming ref --> alt 
But if we think of there being bidirectionality, then how Riley has it makes sense to me. Lines 222 thought 226 would be flagging to G->A as a weak to strong (by backwards). 
However, I do think I preference it being broken up into two separate functions.

Reply:
I agree with Craig and Chelsea. Either change the name or make 2 separate functions.

Reply:
Name changed in most recent commit.

Reply:
I don't see this name change reflected here.  Maybe that commit still needs a push to github?

Reply:
The name here hasn't changed. This function is actually just "WeakToStrong" and a new function has been added to cover the case "WeakToStrongOrStrongToWeak". The most recent commit returned the "strongtoweak" portion of this function and implemented that elsewhere. Should be in line with the comments in the current commit